CONTRIBUTING.md

```diff
@@ -45,6 +45,41 @@ Before you can start contributing to the CyberArk Secrets Provider for Kubernete
     
     b. For detailed setup instructions for OSS, see [CyberArk Secrets Provider for Kubernetes for OSS](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_deployApplicationsConjur-k8s-Secrets.htm).
     
+### Deploy a Local Dev Environment
+
+You can now deploy a local development environment for Kubernetes using [Docker Desktop](https://www.docker.com/products/docker-desktop). Docker Desktop provides a convenient way to deploy and develop from your machine against a cluster deployed locally. 
+To enable this, perform the following:
+
+1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`
+
+1. Run `./bin/start --dev --gke`, appending `--oss` or `--dap` according to the environment that needs to be deployed 
+
+1. If a cluster is already locally deployed, run `./bin/start --dev --reload`. 
+The reload script will build your local changes and redeploy them to the local Secrets Provider K8s cluster.
+
+Currently the development environment only supports deployment to Kubernetes and *not* Openshift.
+
+#### Helpful hints
```

Comment:
i think that this should go higher. i went step by step and only then got to the hints.

Reply:
I removed "helpful hints" because they are basically requirements

Reply:
TODO add how to delete deployments